File: DayOne/Java/LevelOne/NonPerfectSquareArray.java
public class NonPerfectSquareArray{

    public static double [] nonPerfectSquareCheck(int [] array){

    double [] newArray = new double [array.length];

    for (int indexCount = 0; indexCount < array.length; indexCount++){

    double numberSquareRoot = Math.sqrt(array[indexCount]);

    if (numberSquareRoot % 1 != 0) newArray[indexCount] = -1.0;
      
    else newArray[indexCount] = array[indexCount];  
            
    }

    return newArray;
    }
}

File: DayOne/JavaScript/LevelOne/non-perfect-square-array.js
export const nonPerfectSquareCheck = (array)=>{

    let newArray = Array(array.length).fill(0);

    for (let indexCount = 0; indexCount < array.length; indexCount++){

    let numberSquareRoot = Math.sqrt(array[indexCount]);

    if (numberSquareRoot % 1 != 0) newArray[indexCount] = -1;
      
    else newArray[indexCount] = array[indexCount];  
         
    }

    return newArray;
}
